lua/lvim/core/autocmds.lua

```diff
@@ -180,16 +180,23 @@ function M.toggle_format_on_save()
 end
 
 function M.enable_reload_config_on_save()
-  local user_config_file = require("lvim.config"):get_user_config_path()
+  local pattern = get_config_dir() .. "/*.lua"
+
+  -- add symlinked path to the pattern
+  local linked_config_lua = vim.fn.resolve(join_paths(get_config_dir(), "config.lua"))
+  if linked_config_lua ~= join_paths(get_config_dir(), "config.lua") then
+    local linked_config_dir = linked_config_lua:sub(1, #linked_config_lua - #"config.lua")
+    pattern = pattern .. "," .. linked_config_dir .. "*.lua"
+  end
```

Comment:
complexity out-weighs the benefit, those that use symlinks can specify the autocmd manually, since `require("lvim.config"):reload()` is already public.

```suggestion
```